views/searchResults.ejs

```diff
@@ -0,0 +1,48 @@
+<%- include('partials/header') -%>
+  <%- include('partials/navbar') -%>
+    <div class="flex flex-row justify-between">
+      <div
+        class="flex flex-col md:px-16 md:pb-16 sm:w-11/12 lg:w-4/6 bg-white/50 mx-auto px-2 shadow-xl shadow-max-yel-green-500/50">
+        <div class="flex">
+          <h2
+            class="my-4 pb-2 text-4xl w-full font-bold text-dark-slate-gray bg-gradient-to-r from-base-100 border-b-4 border-forest-green">
+            Search results for '<%= userEnteredSearchTerm %>'
+          </h2>
+        </div>
+
+        <% if(searchResults.length===0) { %>
+          <div class="flex h-screen justify-center">
+            <p class="text-center p-[8%] w-3/4 h-[300px] text-dark-slate-gray text-1xl font-bold bg-white/50">
+              It looks like there were not any matches found! Add a recipe of your own to fill this gap or try a
+              different search.
+            </p>
+            <div>
+              <% } else { %> <%- include('partials/searchResultsCard') -%>
```

Comment:
See other suggestion/comment about DRYing code by reusing the `recipeCard.ejs` file here.